migration/src/main/resources/reference.conf

```diff
@@ -0,0 +1,25 @@
+akka.persistence.r2dbc.migration {
+
+  # Akka Persistence plugin to migrate from.
+  # You must also define plugin specific configuration
+  # and application specific serializers for events and snapshots.
+  source {
+    journal-plugin-id = "jdbc-journal"
```

Comment:
I think this setting is only used in tests, so maybe we can remove it to avoid confusion

Reply:
thx, removed